app/ai/voice/agents/breeze_buddy/agent/__init__.py

```
@@ -273,6 +290,58 @@ async def _handle_post_greeting_idle(self, user_idle_config) -> None:
+                    {"name": "Call SID", "value": self.call_sid or "unknown"},
+                ],
+                sections=[
+                    {
+                        "title": "What Happened",
+                        "text": "TTS failed mid-call. Call could not continue.",
+                    }
+                ],
+                fallback_text=f"TTS failed, call ended — {self.call_sid or 'unknown'}",
+                tag_users=tag,
+            )
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)